.github/workflows/go-build.yml

```diff
@@ -0,0 +1,61 @@
+name: Go
+
+on:
+  push:
+    branches: ["main"]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
+          cache: true
+
+      - name: Build RPi
+        run: go build -v -o adif-multitool-rpi ./adifmt
+        env:
+          GOOS: linux
+          GOARCH: arm
+          GOARM: 5
```

Comment:
If I'm reading https://en.wikipedia.org/wiki/Raspberry_Pi#Specifications right, isn't 6 the minimal Raspberry Pi ARM version?

Would it make sense to produce arm and amd64 Linux builds, rather than labeling it specifically for RPi, while ensuring the `GOARM` variable is set to ensure it works on the Pi?

Reply:
I feel like I had issues in the distant past. All just copy and paste from oneoffs I have done in the past.